Recipes/JsonConfiguration/CompactJsonSerializer.cs

```diff
@@ -0,0 +1,88 @@
+    /// and where the caller does not want to rely on <see cref="JsonConvert.DefaultSettings"/>
+    /// </summary>
+#if !RecipesProject
+    [System.Diagnostics.DebuggerStepThrough]
+    [System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage]
+    [System.CodeDom.Compiler.GeneratedCode("Spritely.Recipes", "See package version number")]
+#endif
+    internal static partial class CompactJsonSerializer
+    {
+        /// <summary>
+        /// Deserializes the JSON to the specified .NET type.
+        /// </summary>
+        /// <typeparam name="T">The type of the object to deserialize to.</typeparam>
+        /// <param name="value">The JSON to deserialize.</param>
+        /// <returns>The deserialized object from the JSON string.</returns>
+        public static T DeserializeObject<T>(string value) where T : class
+        {
+            var result = JsonConvert.DeserializeObject<T>(value, JsonConfiguration.CompactSerializerSettings);
+            return result;
+        }
+
+        /// <summary>
+        /// Deserializes the JSON to a .NET object.
+        /// </summary>
+        /// <param name="value">JSON to deserialize.</param>
+        /// <returns>The deserialized object from the JSON string.</returns>
+        public static object DeserializeObject(string value)
+        {
+            var result = JsonConvert.DeserializeObject(value, JsonConfiguration.CompactSerializerSettings);
+            return result;
+        }
+
+        /// <summary>
+        /// Deserializes the JSON to the specified .NET type.
+        /// </summary>
+        /// <param name="value">The JSON to deserialize.</param>
+        /// <param name="type">The Type of object being deserialized.</param>
+        /// <returns>The deserialized object from the JSON string.</returns>
+        public static object DeserializeObject(string value, Type type)
+        {
+            var result = JsonConvert.DeserializeObject(value, type, JsonConfiguration.CompactSerializerSettings);
+            return result;
+        }
+
+        /// <summary>
+        /// Serializes the specified object to a JSON string.
+        /// </summary>
+        /// <param name="value">The object to serialize.</param>
+        /// <returns>A JSON string representation of the object.</returns>
+        public static string SerializeObject(object value)
+        {
+            var result = JsonConvert.SerializeObject(value, JsonConfiguration.CompactSerializerSettings);
+            return result;
+        }
+
+        /// <summary>
+        /// Serializes the specified object to a JSON string using a type.
+        /// </summary>
+        /// <param name="value">The object to serialize.</param>
+        /// <param name="type">The type of the value being serialized. This parameter is used when TypeNameHandling is Auto to write out the type name if the type of the value does not match. Specifing the type is optional.</param>
```

Comment:
Do we specify what TypeNameHandling is? If not, what is the default? Because depending upon what we pick for a particular serializer settings and how this method would behave under that setting, it may make sense to drop this method from some or all of these types (or maybe not).